jenkins_rock/rockcraft.yaml

```diff
@@ -36,12 +36,13 @@ parts:
       - libfontconfig1
       - default-jre-headless
     build-environment:
-      - JENKINS_VERSION: 2.403
+      - JENKINS_VERSION: 2.387
+      - JENKINS_PLUGIN_MANAGER_VERSION: 2.12.11
     override-build: |
       mkdir -p ${CRAFT_PART_INSTALL}/{srv/jenkins/,etc/default/jenkins/}
-      chown 2000:2000 ${CRAFT_PART_INSTALL}/srv/jenkins
       # Use jenkins war rather than apt install for easier Jenkins version control.
       cd ${CRAFT_PART_INSTALL}/srv/jenkins
+      curl -sLO https://github.com/jenkinsci/plugin-installation-manager-tool/releases/download/${JENKINS_PLUGIN_MANAGER_VERSION}/jenkins-plugin-manager-${JENKINS_PLUGIN_MANAGER_VERSION}.jar
       curl -sLO https://mirrors.jenkins.io/war/${JENKINS_VERSION}/jenkins.war
```

Comment:
Shouldn't we verify the war signature ?
See: https://www.jenkins.io/download/verify/

Reply:
Added, thank you!